spec/requests/dashboard_spec.rb

```diff
@@ -7,11 +7,14 @@
 
   context 'when dashboard page is displayed' do
     let!(:edition) { create(:edition, :with_participants, count: 5) }
+    let!(:position) { create(:position) }
```

Comment:
We may need to create multiple positions and multiple participants in order to reflect real data.